images/output/01-full/Dockerfile

```diff
@@ -10,6 +10,10 @@ ARG REPO
 ARG TAG
 FROM ${REPO}/k3os-kernel:${TAG} as kernel
 
+ARG REPO
+ARG TAG
+FROM ${REPO}/k3os-qemu:${TAG} as qemu
```

Comment:
Exclusion of `arn64` from the qemu part of the build is causing it to fail. See https://github.com/rancher/k3os/pull/434/files#r410665601